ipamutils/utils.go

```diff
@@ -48,3 +63,86 @@ func initGranularPredefinedNetworks() []*net.IPNet {
 	}
 	return pl
 }
+
+// InitAddressPools allows to initialize the local and global scope predefined
+// address pools to the desired values. It fails is invalid input is passed
+// or if the predefined pools were already initialized.
+func InitAddressPools(list []*PredefinedPools) error {
+	var (
+		localPools  []*net.IPNet
+		globalPools []*net.IPNet
+		done        bool
+	)
+
+	for _, p := range list {
+		if p == nil {
+			continue
+		}
+		_, b, err := net.ParseCIDR(p.Base)
+		if err != nil {
+			return fmt.Errorf("invalid base pool %q: %v", p.Base, err)
+		}
+		ones, _ := b.Mask.Size()
+		if p.Size <= 0 || p.Size < ones {
+			return fmt.Errorf("invalid pools size: %d", p.Size)
+		}
+		switch p.Scope {
+		case "", "local":
```

Comment:
Lets make the scope mandatory rather than assuming the default is local. Because when we support the user provided range for swarmkit the default should be global. Making it explicit would be better instead.